### PR TITLE
fix: windows, ubuntu: emojiPicker glitches when opening

### PR DESCRIPTION
**Pull request by PaulDremanovich**

### CODE REVIEW

**Comment by PaulDremanovich** [comment]
[UI: Add emoji and move attach to the input field (copied from iOS)](https://trello.com/c/x5HC9Uyj/426-ui-add-emoji-and-move-attach-to-the-input-field-copied-from-ios)

File: src/components/EmojiPicker.vue [hunk 1]
```diff
@@ -64,13 +65,21 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
+.emoji-picker.is-dark {
```
Comment (bludnic) [inline src/components/EmojiPicker.vue:68]: try to use `.theme--dark {}` and `.theme--light {}`. See example: https://github.com/Adamant-im/adamant-im/blob/6c0b5cba4ec14593494bdc73ab75085069f111fb/src/components/PartnerInfo.vue#L143

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/components/EmojiPicker.vue [hunk 2]
```diff
@@ -64,18 +64,28 @@ export default defineComponent({
...
+@import 'vuetify/settings';
+@import '../assets/styles/settings/_colors.scss';
 .emoji-picker {
   border-radius: 8px;
 
   em-emoji-picker {
     width: 264px;
     height: 264px;
     border-radius: 8px;
+    border-color: map-get($adm-colors, 'light-theme-border');
```
Comment (bludnic) [inline src/components/EmojiPicker.vue:76]: This section must contain only common styles for both themes. Don't put here styles like: border-color, color and background-color, etc.

Move the `border-color` to `.v-theme--light {...}`

File: src/assets/styles/settings/_colors.scss [hunk 3]
```diff
@@ -13,5 +13,7 @@ $adm-colors: (
   'secondary2-transparent': rgba(245, 245, 245, 0.3),
   'secondary2-slightly-transparent': rgba(245, 245, 245, 0.13),
   'grey': #9e9e9e,
-  'grey-transparent': rgba(255, 255, 255, 0.7)
+  'grey-transparent': rgba(255, 255, 255, 0.7),
+  'dark-theme-border': rgba(255, 255, 255, var(--v-border-opacity)),
+  'light-theme-border': rgba(0, 0, 0, var(--v-border-opacity))
```
Comment (bludnic) [inline src/assets/styles/settings/_colors.scss:18]: Remove these color constants. Write them directly in EmojiPicker.vue

**Review by bludnic — CHANGES_REQUESTED** [review]

**Review by adamant-al — APPROVED** [review]

File: src/components/EmojiPicker.vue [hunk 4]
```diff
@@ -64,18 +64,31 @@ export default defineComponent({
...
 
     #root {
       border-top-left-radius: 0;
       border-top-right-radius: 0;
     }
   }
 }
+.v-theme--dark {
+  em-emoji-picker {
+    border-color: $dark-theme-border-color;
```
Comment (bludnic) [inline src/components/EmojiPicker.vue:91]: No need in defining a constant. It is used only in one place. Use the color value directly:

```suggestion
    border-color: rgba(map-get($shades, 'black'), var(--v-border-opacity))
```

File: src/components/EmojiPicker.vue [hunk 5]
```diff
@@ -64,18 +64,31 @@ export default defineComponent({
...
+$light-theme-border-color: rgba(map-get($shades, 'black'), var(--v-border-opacity));
+
 .emoji-picker {
   border-radius: 8px;
 
   em-emoji-picker {
     width: 264px;
     height: 264px;
     border-radius: 8px;
+    border-color: $light-theme-border-color;
```
Comment (bludnic) [inline src/components/EmojiPicker.vue:79]: Move `border-color` to .v-theme--light

**Review by bludnic — CHANGES_REQUESTED** [review]

**Review by bludnic — APPROVED** [review]

**Comment by adamant-al** [comment]
@bludnic Can we merge?
I want it in the current release.

**Review by RealGoodProgrammer — APPROVED** [review]

**Pull request merged**